src/components/ThreeDEditor.js

```diff
@@ -339,8 +323,15 @@ export class ThreeDEditor extends React.Component {
         if (this.state.isThreejsEditorModalShown) {
             return <ThreejsEditorModal
                 show={this.state.isThreejsEditorModalShown}
-                onHide={this.ToggleThreejsEditorModal}
-                materials={[this.props.material]}
+                onHide={(material) => {
+                    this.setState({
```

Comment:
Let's move this to a handler function above

Reply:
Done.